services/webnn/ort/graph_builder_ort.h

```diff
@@ -94,17 +94,24 @@ class GraphBuilderOrt {
   const mojom::Operand& GetOperand(uint64_t operand_id);
   std::string GetOperandName(uint64_t operand_id);
 
-  // Some initializers must be uploaded to raw data, e.g. Reshape op needs
-  // parameter *shape* as raw data to do shape inference.
+  // Create a new initializer copied into graph.
+  uint64_t NewInitializer(base::span<const uint32_t> shape,
```

Comment:
Why do we need this method given we already have the following two versions? Is it just used internally?

You may want to coordinate with @mingmingtasd , Mingming is also refactoring this method.

Reply:
Yes, in https://github.com/shiyi9801/chromium/pull/50.

Reply:
Rebased on Mingming's latest code.

Reply:
> Why do we need this method given we already have the following two versions? Is it just used internally?

Yes, I wanted to keep these two methods at the beginning, but function `CreateInitializerAsExternalData` may not be used for the time being. Let's move on and add them if we need them.
And please take another look, thanks!

Reply:
> > but function CreateInitializerAsExternalData may not be used for the time being.
> 
> Why? Do you mean we only need to create initializers as raw data?

I mean, the operators we have implemented currently do not necessarily  create initializers as external data. Once we encounter the need to use external data to reduce copying, we will add this method. WDYT？

Reply:
Did you mean the use case of converting WebNN ops' attributes to ONNX ops' initializers? I assume constant operand would still be created as initializers using external data, correct?